rllib/models/catalog.py

```diff
@@ -190,6 +190,12 @@
     # outputs floating bias variables instead of state-dependent. This only
     # has an effect is using the default fully connected net.
     "free_log_std": False,
+    # Whether to clip the log standard deviation when using a Gaussian (or any
+    # other continuous control distribution). This can stabilize training and avoid
+    # very small or large log standard deviations leading to numerical instabilities
+    # which can turn network outputs to `nan`. The default is infinity, i.e. no
+    # clipping.
+    "log_std_clip_param": float("inf"),
```

Comment:
Ah, ok, here it is. But still, as we'll soon get rid of the old stack model config dict, we should be defensive against users bringing their own `model_config_dict` to BC or other algos.